java/templates/Dockerfile

```diff
@@ -50,6 +50,9 @@ RUN curl https://archive.apache.org/dist/maven/maven-3/{{= mavenVersion }}/binar
             userGroupMode: "root",
             version: fp.config.base.lib.version.jolokia }) }}
 
+# Install rsync
+RUN yum install -y rsync
```

Comment:
For the RHEL image, I don't think this is required as rsync already seems to be [installed](https://github.com/jboss-container-images/jboss-base-image/blob/1e338115b0316c8e1c1e54e0e0dfa36135d399c5/image.yaml#L16) in the recent JDK base images.

```
$ docker run -ti --rm jboss/openjdk18-rhel7:1.0-11 bash
[jboss@43fd7f8c3d4b ~]$ which rsync
/usr/bin/rsync
```
You should be able to add some conditional logic around the `yum install` command within the template to see if we're generating the RHEL image or not. E.g `{{? fp.param.base === "rhel"}}`.

Reply:
why would we want it only for rhel image?

Reply:
and rsync was not available with the image pointed by Dhiraj to use  https://hub.docker.com/r/fabric8/s2i-java/ https://issues.jboss.org/browse/OSFUSE-548?focusedCommentId=13459061&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13459061
do you know what is the difference between the image you are using and the one pointed by Dhiraj?

Reply:
My bad, I meant only add rsync for the != "rhel" case.

Reply:
ok got it,
rsync is available in the rhel base image one but not on the community one

I will update it